### PR TITLE
Removes employment contracts

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request
Removes employment contracts cabinet from the lawyer offices and gives them a normal filing cabinet

## Why It's Good For The Game
Whoever it was on tg that did the "NT owns your soul" thing is fucking stupid

## Changelog
:cl:
del: Removed the employment contract cabinets
/:cl:

### CODE REVIEW

**Comment by ShadeAware** [comment]
I mean, those contracts are used against Devils, and could provide some roleplay opportunities.  Might be better to just change the wording on the actual contract itself tbh.

**Comment by Commandersand** [comment]
pretty realistic, thank you mega corporations

File: _maps/map_files/KiloStation/KiloStation_Skyrat.dmm [hunk 1]
```diff
@@ -16517,7 +16517,6 @@
 /turf/open/floor/engine,
 /area/security/nuke_storage)
 "aAC" = (
-/mob/living/simple_animal/pet/roro/roboticsroro,
```
Comment (improvedname) [inline _maps/map_files/KiloStation/KiloStation_Skyrat.dmm:16520]: Undocumented change

File: _maps/map_files/KiloStation/KiloStation_Skyrat.dmm [hunk 2]
```diff
@@ -73072,7 +73071,6 @@
 /obj/structure/cable,
 /obj/machinery/light/small,
 /obj/effect/decal/cleanable/dirt,
-/mob/living/simple_animal/pet/roro,
```
Comment (improvedname) [inline _maps/map_files/KiloStation/KiloStation_Skyrat.dmm:73075]: Undocumented change

File: _maps/map_files/KiloStation/KiloStation_Skyrat.dmm [hunk 3]
```diff
@@ -87774,7 +87772,6 @@
 	},
 /obj/item/reagent_containers/food/drinks/ale,
 /obj/structure/table/wood,
-/obj/item/instrument/eguitar,
```
Comment (improvedname) [inline _maps/map_files/KiloStation/KiloStation_Skyrat.dmm:87777]: Undocumented change

**Comment by ForrestWick** [comment]
Stuff fucked up, can't fix it atm. Closing until I got time and better wifi

**Pull request closed without merging**